op-node/rollup/derive/l1_retrieval_test.go

```diff
@@ -86,10 +86,11 @@ func TestL1RetrievalReset(t *testing.T) {
 	dataSrc := &MockDataSource{}
 	a := testutils.RandomBlockRef(rng)
 	l1Cfg := eth.SystemConfig{
-		BatcherAddr: common.Address{42},
+		BatcherAddr:        common.Address{42},
```

Comment:
Why are there so many spaces between `BatcherAddr` and `common.Address{42}`.

Reply:
`gofmt` does this alignment.